ignore-escape-char.patch

```diff
@@ -0,0 +1,75 @@
+diff --git a/System/Metrics/Json.hs b/System/Metrics/Json.hs
```

Comment:
@Naveenaidu  You probably didn't want to commit this file

Reply:
Yes, Snap! Sorry will delete this